core/src/main/java/org/infinispan/stats/impl/ClusterCacheStatsImpl.java

```diff
@@ -257,6 +270,29 @@ public long getMisses() {
       return getStatAsLong(MISSES);
    }
 
+   @ManagedAttribute(description = "Cluster wide approximate number of entries in the cache, including passivated entries and ignoring copies",
```

Comment:
```suggestion
   @ManagedAttribute(description = "Approximate number of entries in the cache across the cluster, including passivated entries but not replicas.",
```

Reply:
Actually the description was wrong here, this stat does include replicas :)